examples/rollbar-spring-webmvc/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+
```

Comment:
I think that we could add here some rollbar properties needed for configuration like the access token, the environment, the code version...
Then, to have access to them in the code we could create class that uses https://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/context/properties/ConfigurationProperties.html to load them.

Reply:
now i am using the application.properties